packages/visx-xychart/test/components/XYChart.test.tsx

```diff
@@ -21,24 +16,18 @@ describe('<XYChart />', () => {
     expect(XYChart).toBeDefined();
   });
 
-  it('should render a ParentSize if width or height is not provided', () => {
-    const wrapper = mount(
-      <XYChart {...chartProps} width={undefined}>
-        <rect />
-      </XYChart>,
+  it('should render with parent size if width or height is not provided', () => {
+    const { getByTestId } = render(
+      <div style={{ width: '200px', height: '200px' }} data-testid="wrapper">
+        <XYChart {...chartProps} width={undefined} data-testid="rect">
+          <rect />
+        </XYChart>
+      </div>,
     );
-    expect(wrapper.find(ParentSize)).toHaveLength(1);
-  });
 
-  it('should render DataProvider, EventEmitterProvider, and TooltipProvider if not available in context', () => {
-    const wrapper = mount(
-      <XYChart {...chartProps}>
-        <rect />
-      </XYChart>,
-    );
-    expect(wrapper.find(DataProvider)).toHaveLength(1);
```

Comment:
not sure how best to test this one since context is an implementation detail kind of by definition.

Reply:
yea, because when this component gets rendered, you don't really see a DataProvider, EventEmitterProvider or TooltipProvider in your DOM anyways.